integration/query_test.go

```diff
@@ -834,117 +834,117 @@ func TestQueryCommandUnsafeLimitPushDown(t *testing.T) {
 		sort    bson.D // optional, nil to leave sort unset
 		optSkip *int64 // optional, nil to leave optSkip unset
 
-		len                 int                 // expected length of results
-		queryPushdown       resultPushdown      // optional, defaults to noPushdown
-		unsafeLimitPushdown bool                // optional, set true for expected pushdown for limit
-		err                 *mongo.CommandError // optional, expected error from MongoDB
-		altMessage          string              // optional, alternative error message for FerretDB, ignored if empty
-		skip                string              // optional, skip test with a specified reason
-		failsForSQLite      string              // optional, if set, the case is expected to fail for SQLite due to given issue
+		len            int                 // expected length of results
+		filterPushdown resultPushdown      // optional, defaults to noPushdown
+		limitPushdown  bool                // optional, set true for expected pushdown for limit
```

Comment:
Why is that `bool` and not `resultPushdown`?

Reply:
fixed